scripts/dockerhost/setlocalns.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+scriptdir=`dirname "$BASH_SOURCE"`
+HOST_SCRIPTDIR=$HOST_GOPATH/src/github.com/contiv/netplugin/scripts/dockerhost
+pid=$($scriptdir/host-sh $HOST_SCRIPTDIR/docker-pid $(hostname))
+FILES=$($scriptdir/host-sh ls /sys/class/net/)
```

Comment:
is the path '/sys/class/net' os specific i.e. can it differ between say ubuntu and centos host oses? Just wondering if this puts an assumption on where the DIND environment is deployed. If so may be we should put it in a comment here.